R/extremes-data.R

```diff
@@ -1,13 +1,29 @@
+#' Get Extremes Report constants
+#' @description list of constants used by extremes report
+getExtremesConstants <- function() {
+  return(list(
+          PRIMARY_HEADER_PREFIX="Primary series",
+          UPCHAIN_HEADER_PREFIX="Upchain series"
+          ))
+}
+
 #' Create a Flat Text "extremes table" Type Output Table
 #' 
-#' @param rawData An extremes report JSON string.
+#' @param reportObject An extremes report JSON string.
 #' @importFrom dplyr mutate
 #' @return string table
 #' @export
```

Comment:
Bah, sorry, I thought you were asking if it should be included. Will take out on next pr.